ac_runtime-api/modules/parachain.adoc

```diff
@@ -305,3 +332,83 @@ Arguments::
 
 Return::
 * An array of inbound HRMP messages (<<defn-inbound-hrmp-message>>).
+
+
+==== `ParachainHost_on_chain_votes`
+
+Returns disputes relevant from on-chain, backing votes, and resolved disputes. 
+
+Arguments::
+* None
+
+Return::
+* An _Option_ type which can contain the Scraped OnChainVotes data (<<defn-scraped-on-chain-vote>>). 
```

Comment:
* `"An _Option_  value (<<defn-option-type>>) ..."`
* Custom (Rust) type names should be italic, i.e. `"_OnChainVotes_"`. Alternatively (or preferably), just say `"... can contain the scraped on-chain votes data..."`.